packages/react-dom/src/client/ReactDOMFiberComponent.js

```diff
@@ -378,6 +378,12 @@ export function createElement(
       // See discussion in https://github.com/facebook/react/pull/6896
       // and discussion in https://bugzilla.mozilla.org/show_bug.cgi?id=1276240
       domElement = ownerDocument.createElement(type);
+      // Make sure that `select` has `multiple` attribute set to appropriate state before appending options
+      // To prevent first option be initialy made selected
+      // see more details in https://github.com/facebook/react/issues/13222
```

Comment:
This isn't perfect, but could you update this comment to read like:

```javascript
// Normally attributes are assigned in `setInitialDOMProperties`, however the `multiple`
// attribute on `select`s needs to be added before `option`s are inserted. This prevents 
// a bug where the `select` does not scroll to the correct option because singular 
// `select` elements automatically pick the first item.
// See https://github.com/facebook/react/issues/13222
```

@gaearon Could this be clearer?